README.md

```diff
@@ -42,3 +42,21 @@ We aim to build a library for C++ 17 or above for big integer support
 - CMake 3.16.8 or above
 - GCC 9
 - C++ 17
+
+
+#### Chidu2000 , 04-10-2020
```

Comment:
no need to mention these things in the `readme.md`, we are at the initial stage of development, once we are at good level with some features implemented we will setup seprate documentaion for this.